src/ontology/mondo-edit.obo

```diff
@@ -230650,6 +230658,7 @@ def: "Autosomal recessive limb-girdle muscular dystrophy type 2I (LGMD2I) is a s
 subset: ordo_disease {source="Orphanet:34515"}
 synonym: "autosomal recessive limb-girdle muscular dystrophy caused by mutation in FKRP" EXACT [MONDO:design_pattern]
 synonym: "FKRP autosomal recessive limb-girdle muscular dystrophy" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+synonym: "LGMD-FKRP related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/